library/ESign/Form/Signable.php

```diff
@@ -118,6 +118,11 @@ public function getData()
         }
 
         $tbl = (isset($excp->tbl) ? $excp->tbl : "form_" . $this->_formDir);
+        // Either change here or revise eye form code to use
+        // table form_eye_mag instead of form_eye_base
+        // Could be as simple as a sed function but for now, do this:
+        if ($tbl =='form_eye_mag') { $tbl = 'form_eye_base'; }
```

Comment:
if this works, then would go with this line. If this is testing well for you, then let me know. Then will bring this fix into master and rel-600 (for a 6.0.0 patch).